datanode/gateway/graphql/schema.graphql

```diff
@@ -545,23 +555,23 @@ type Query {
     pagination: Pagination
   ): MarketDataConnection
 
-  "Get historical balances for an account within the given date range"
-  balanceChanges(
-    "Optional filter to restrict queried accounts to those of a specific asset, party, market or type"
-    filter: AccountFilter
-    "Date range to retrieve historical balances from/to. Start and end time should be expressed as an integer value of nano-seconds past the Unix epoch"
-    dateRange: DateRange
-    "Optional pagination information"
-    pagination: Pagination
-  ): AggregatedBalanceConnection!
-
   "Get ledger entries by asset, market, party, account type, transfer type within the given date range."
   ledgerEntries(
     filter: LedgerEntryFilter
     dateRange: DateRange
     pagination: Pagination
   ): AggregatedLedgerEntriesConnection!
 
+  "List all active liquidity providers for a specific market"
```

Comment:
**nitpick (non-blocking)**: This feels very similar to the existing `liquidityProvisionsConnection` API, so I'm not sure that it's required. I don't object to it being here, but console uses the existing API which has the benefit of also allowing for nesting queries for the party and other fields

Reply:
The liquidity provisions API doesn't provide the following data for the party which has to be extrapolated from the `market_data` events:

- Equity Like Share
- Average Score
- Average Entry Valuation
- Virtual Stake

I probably confused "This data must be available from the API" with "We need an API function that provides this data", but looking at the liquidity provisions API it doesn't return the above data directly, and API users would need to know to retrieve it from Market Data if they wanted it.